webapi/src/main/scala/org/knora/webapi/responders/v1/OntologyResponderV1.scala

```diff
@@ -500,49 +494,48 @@ final case class OntologyResponderV1Live(
     userProfile: UserADM
   ): Task[ResourceTypesForNamedGraphResponseV1] = {
 
+    def getProperties(
+      resInfo: ResourceTypeResponseV1
+    ): IO[InconsistentRepositoryDataException, Vector[PropertyTypeV1]] = ZIO.collectAll(
+      resInfo.restype_info.properties.map { prop =>
+        for {
+          label <-
+            ZIO.fromOption(prop.label).orElseFail(InconsistentRepositoryDataException(s"No label given for ${prop.id}"))
+        } yield PropertyTypeV1(
+          id = prop.id,
+          label = label
+        )
```

Comment:
IMHO we should avoid using named parameters if not necessary. 
In above example `PropertyTypeV1(prop.id,  label)` would reduce visual noise and improve readability.
There are other places in the code you could consider this.

What do you think?

Reply:
I agree. I remove named params whenever I feel it's needed. But we had similar discussion already before, should we improve all code which is marked as changed, only because we change indentations for instance?